lib/matplotlib/dates.py

```diff
@@ -981,16 +981,27 @@ def __call__(self, x, pos=None):
 
 class rrulewrapper:
     """
-    A simple wrapper around a ``dateutil.rrule`` allowing flexible
+    A simple wrapper around a `dateutil.rrule` allowing flexible
     date tick specifications.
     """
     def __init__(self, freq, tzinfo=None, **kwargs):
+        """
+        Parameters
+        ----------
+        freq : {YEARLY, MONTHLY, WEEKLY, DAILY, HOURLY, MINUTELY, SECONDLY}
+            Tick frequency.
+        tzinfo : `datetime.tzinfo`, optional
+            Time zone information. The default is None.
+        **kwargs
+            Additional keyword arguments passed to the `rrule`.
```

Comment:
```suggestion
            Additional keyword arguments are passed to the `dateutil.rrule`.
```
Interestingly, sphinx regenerates the `dateutil.rrule` docs in `matplotlib.dates`:
https://matplotlib.org/stable/api/dates_api.html#matplotlib.dates.rrule (might have to do with that we specify `__all__` (https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-option-automodule-members). If I haven't overlooked anything, that's undesired, because it's not a class we define. Anywy should point the link to the correct original definition.

Figuring out how to to remove the `rrule` docs can be in this PR or left for later as you like.


https://matplotlib.org/stable/api/dates_api.html#matplotlib.dates.rrule

Reply:
I also noted that (which is why it was directly referenced) and was doubtful if that was expected, so glad to get input on it. I'll see if I can rid of it. Good points above! Thanks!